source/declarations.tex

```diff
@@ -5755,6 +5755,8 @@
 list-initialization in a direct-initialization context is called
 \defn{direct-list-initialization} and list-initialization in a
 copy-initialization context is called \defn{copy-list-initialization}.
+Direct-initialization that is not direct-list-initialization is called
```

Comment:
This seems more straightforward:

```suggestion
Direct-initialization that is not list-initialization is called
```

Reply:
I was thinking about that, too, and thought that direct-list-iniitialization is more immediately in memory, but it can go either way. Sure.